Sources/Testing/Expectations/Expectation.swift

```diff
@@ -10,9 +10,9 @@
 
 /// A type describing an expectation that has been evaluated.
 public struct Expectation: Sendable {
-  /// The source code of the expression evaluated by this expectation, if
-  /// available at compile time.
-  public var sourceCode: SourceCode?
+  /// The expression evaluated by this expectation, if available at compile
+  /// time.
+  public var evaluatedExpression: Expression?
```

Comment:
The description of this property now feels a bit odd: Since `Expression` now contains both the source code of an expression and the runtime value (if evaluated) of said expression, the "if available at compile time" may be confusing since one of those (the runtime value) can't possibly be known at compile time.

Can this wording be clarified? (And any edits propagated down to the corresponding `Snapshot` property below)

Reply:
We can remove the subordinate clause ("if …")?

Reply:
Removed.